pkg/cephfs/controllerserver.go

```diff
@@ -235,6 +235,7 @@ func (cs *ControllerServer) DeleteVolume(ctx context.Context, req *csi.DeleteVol
 		// or partially complete (subvolume and imageOMap are garbage collected already), hence
 		// return success as deletion is complete
 		if _, ok := err.(util.ErrKeyNotFound); ok {
+			klog.Warningf(util.Log(ctx, "Failed to volume options for %s: %v"), string(volID), err)
```

Comment:
(nit) error message needs some massaging "failed to find backing volume %s for volume ID %v" (same in the rbd controllerserver file as well)

(nit) here and in some other places, remove capitalization of leading message string

Reply:
This is a log, why to remove capitalization?

Reply:
I checked the current codebase, in most of the cases we are logging as small case but it's not required always see https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Technically the log message is also prepended with a "Req-ID: XXX : <message>" so capitalization rules as per the error message link applies (but can be argued otherwise as well). It also makes sense to follow a single style across the code, which I assumed was not capitalization. Anyway, I am not spending more time on this nit...

Reply:
Yes we  need to follow single code side in the codebase, will revisit the codebase to check it and will send a PR